.github/workflows/ci.yml

```diff
@@ -66,6 +66,32 @@ jobs:
           -DS3_PRESIGNED_PUT_URL="put-url"
       - name: Build System Tests
         run: make -C build/ help | grep system | tr -d '. ' | xargs make -C build/
+  build-check-install:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone This Repo
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Configure CMake build
+        run: |
+          mkdir installed-headers-dir installed-libs-dir
+          curl https://cmake.org/files/v3.2/cmake-3.2.0-Linux-x86_64.tar.gz -o cmake.tar.gz
+          tar -xf cmake.tar.gz
+          mkdir build && cd build
+          ../cmake-3.2.0-Linux-x86_64/bin/cmake .. \
+          -DBUILD_DEMOS=0 -DBUILD_TESTS=0 \
+          -DCSDK_LIB_INSTALL_PATH="../installed-headers-dir" -DCSDK_LIB_INSTALL_PATH="../installed-headers-dir"
+      - name: Install libraries
+        run: |
+          sudo make -C build/ install
```

Comment:
(Nit) Is `sudo` required for installing in the non-system location?

Reply:
Not necessary but I did this because `sudo` is also used in the README